arangod/GeneralServer/HttpCommTask.cpp

```diff
@@ -386,7 +386,7 @@ void HttpCommTask<T>::processRequest() {
   this->_protocol->timer.cancel();
   
   {
-    LOG_TOPIC("6e770", DEBUG, Logger::REQUESTS)
+    LOG_TOPIC("6e770", INFO, Logger::REQUESTS)
```

Comment:
I think this should go back to log level DEBUG. 
Why? Because we tell people that they can use `--log.level requests=info` to log all requests easily. Right now this will log one line per request, but with the above change it will log multiple lines per request. So let's please revert to debug here.
```suggestion
    LOG_TOPIC("6e770", DEBUG, Logger::REQUESTS)
```